angular.json

```diff
@@ -416,6 +416,15 @@
             }
           }
         },
+        "build-schematics": {
+          "builder": "@nrwl/workspace:run-commands",
+          "options": {
+            "commands": [
+              "cd libs/ui/autocomplete && yarn build && cd ../../.."
```

Comment:
Probably not worth changing now, but for future reference you can do `cd -` to go to the previous directory